src/components/sidebar/SidebarNavigation/index.tsx

```diff
@@ -50,6 +53,16 @@ const Navigation = (): ReactElement => {
     })
   }, [chain, isBlockedCountry])
 
+  const enabledNavItems = useMemo(() => {
+    return visibleNavItems.filter((item) => {
+      if (!safe.deployed && undeployedSafeBlockedRoutes.includes(item.href)) {
+        return false
+      }
+
+      return true
```

Comment:
```suggestion
      return !(!safe.deployed && undeployedSafeBlockedRoutes.includes(item.href))
```

nit

Reply:
I will invert the condition to remove the double negation and adjust :)